backend/src/bundles/users/user.service.ts

```diff
@@ -39,8 +39,10 @@ class UserService implements Service {
                 passwordHash: hash, // TODO
             }),
         );
+        const { id, email } = user.toObject();
+        const token = await tokenService.createToken(id);
 
-        return user.toObject();
+        return { 'id': id, 'email': email, 'token': token };
```

Comment:
You can use property shorthand here as well

Reply:
Done. Thank you